xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +212,70 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc. Wireless Phone (.*)" certainty="0.9">
```

Comment:
Include $ end of string

Reply:
OK, I can do that. My reasoning was that no matter what comes after, it's still a wireless phone, even if the model is 7925 or 7925G or whatever  (saves us from having to add specific models in the future)

Reply:
While tighter regex sometimes require updates, unbounded matches may result in unexpected matches and captures as well as possible performance issues (see #368).

Reply:
>  My reasoning was that no matter what comes after, it's still a wireless phone, even if the model is 7925 or 7925G 

It's actually a bit more of an issue than that. This may silently match a new variant that is of interest to us in the future. 

For example, `Cisco Systems Inc. Wireless Phone 990x running NewPhoneOS 10.1` would be matched but we'd want to know about this new banner so we can add specific coverage for it and assert `os.*` attributes.